### PR TITLE
Add pronoun text

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR adds text from the DIB office to the user profile settings page. 

Resolves TLT-3999.

### CODE REVIEW

File: js/pronoun_text.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <a href=\"https://dib.harvard.edu/gender-pronouns\">\
+    https://dib.harvard.edu/gender-pronouns</a>.\
+    ");
+
+    $( "#pronouns" ).parent().parent().children("th").css("vertical-align", "top");
+
+  }
+}
+
+$(document).ready(initAddPronounText);
```
Comment (elliottyates) [inline js/pronoun_text.js:29]: Q: is the [branding page I found on k459](https://wiki.harvard.edu/confluence/pages/viewpage.action?pageId=157484597) up-to-date, per deploying these? Also, should the global JS files be part of the code committed, per the [README](https://github.com/harvard-canvas-branding/canvas-branding-global#readme)?

File: js/pronoun_text.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+function initAddPronounText() {
+  var windowUrl = window.location.pathname;
+  var onProfileSettingsPage = (windowUrl.search(/\/profile\/settings/) != -1);
+
+  if (onProfileSettingsPage) {
+    textSpan = $( "#pronouns" ).parent().children().last();
```
Comment (elliottyates) [inline js/pronoun_text.js]: nit: should this be declared with `var` as well?

**Review by elliottyates — APPROVED** [review]
👍

File: js/pronoun_text.js (hunk 2 above)
Comment (cmurtaugh) [reply js/pronoun_text.js]: Ah, yes, probably.

File: js/pronoun_text.js (hunk 1 above)
Comment (cmurtaugh) [reply js/pronoun_text.js:29]: The page on k459 doesn't really talk about deployment, but the readme should be updated.

**Pull request closed without merging**